saleor/graphql/checkout/mutations.py

```diff
@@ -113,7 +113,8 @@ def clean_delivery_method(
         )
 
     valid_methods = checkout_info.valid_delivery_methods
-    return method in valid_methods
+    # TODO: use me to check for price mismatches
```

Comment:
Restore the original version and check if this change is still needed.

Reply:
It isn't actually, thanks